lib/rdf/datatypes/boolean.ex

```diff
@@ -5,12 +5,15 @@ defmodule RDF.Boolean do
 
   use RDF.Datatype, id: RDF.Datatype.NS.XSD.boolean
 
-  import RDF.Literal.Guards
+  import Literal.Guards
+  alias Literal
 
   @type value :: boolean
+  @type input :: value | String.t | number
```

Comment:
Instead of having always `value | input | Literal.t | any` on the input arguments below, I would add `Literal.t | any` here. Similar in the other datatype definitions.

Reply:
No objections, changed.